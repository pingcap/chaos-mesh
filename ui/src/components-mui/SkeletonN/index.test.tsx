import SkeletonN from '.'
import { render } from '@testing-library/react'

describe('<Skeleton />', () => {
  it('matches snapshot', () => {
    const { container } = render(<SkeletonN n={3} />)

    expect(container).toMatchSnapshot()
  })
})
