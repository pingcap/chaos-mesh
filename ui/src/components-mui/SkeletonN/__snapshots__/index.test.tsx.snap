// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Skeleton /> matches snapshot 1`] = `
<div>
  <span
    class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
  />
  <span
    class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
  />
  <span
    class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
  />
</div>
`;
